### PR TITLE
Moved the dimensions to dimens.xml file to improve readability.

### PR DESCRIPTION
**Pull request by codersanjeev**

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```
Comment (shubhamvashisht) [inline .idea/inspectionProfiles/Project_Default.xml]: don't revision this file.

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.3'
```
Comment (shubhamvashisht) [inline build.gradle]: keep it 3.1.2

File: ktoasty/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="image_width">24dp</dimen>
+    <dimen name="image_height">24dp</dimen>
+    <dimen name="margin">8dp</dimen>
+</resources>
```
Comment (shubhamvashisht) [inline ktoasty/src/main/res/values/dimens.xml]: add new line at the end of file.

File: build.gradle (hunk 2 above)
Comment (codersanjeev) [reply build.gradle]: done.

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (codersanjeev) [reply .idea/inspectionProfiles/Project_Default.xml]: why don't add /.idea in.gitignore file then ?

**Comment by shubhamvashisht** [comment]
Thank you @thedevelopersanjeev

**Pull request merged**